src/dk/aau/cs/ds306e18/tournament/model/TieBreaker.java

```diff
@@ -0,0 +1,67 @@
+package dk.aau.cs.ds306e18.tournament.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public enum TieBreaker {
+    // Goal diff -> goals scored -> seed
```

Comment:
It would. But I estimated that it would require more code to build the ability to chain. We could do it in the future if tie breakers get more complicated.